specification/ws/src/main/scripts/org/kaazing/specification/ws/opening/request.header.sec.websocket.protocol/handshake.response.rpt

```diff
@@ -26,7 +26,8 @@ read header "Upgrade" /(?i:websocket)/
 read header "Connection" /(?i:Upgrade)/
 read header "Sec-WebSocket-Key" /(?<key>[a-zA-Z0-9+\/=]{24})/
 read header "Sec-WebSocket-Version" "13"
-read header "Sec-WebSocket-Protocol" "primary, secondary"
+read header "Sec-WebSocket-Protocol" "primary"
+read header "Sec-WebSocket-Protocol" "secondary"
```

Comment:
Why is this needed?  If I recall correctly the comma separated http headers are equivalent to this, logically from http rfc